### ISSUE TITLE
[BootstrapButton]: Text set programmatically is not visible

### ISSUE THREAD

**Issue by shell-software** [Bearded-Hen/Android-Bootstrap#108]
Hi!

Thank you for your library

While using the Library I faced with a strange issue: when I set the **BootstrapButton** text programmatically by calling **setText(String)** it is set (log message shows it), however it is not visible:
![untitled](https://cloud.githubusercontent.com/assets/10495282/9423083/305cb828-48bd-11e5-9144-678239854f0e.png)

I am using fragment with such layout:
```java
<LinearLayout
			xmlns:bootstrapButton="http://schemas.android.com/apk/res-auto"
			xmlns:tools="http://schemas.android.com/tools"
			android:layout_height="match_parent"
			android:layout_width="match_parent"
			android:padding="16dp"
			android:layout_gravity="center"
			android:gravity="center"
			android:orientation="vertical"
			android:background="@color/Color.Material.White"
			>
	<com.beardedhen.androidbootstrap.BootstrapButton
				android:id="@+id/fragment_rate_button_good"
				android:layout_width="match_parent"
				android:layout_height="wrap_content"
				android:layout_margin="16dp"
				bootstrapButton:bb_type="success"
				bootstrapButton:bb_roundedCorners="true"
				bootstrapButton:bb_size="large"
				bootstrapButton:bb_icon_left="fa-check"
				android:textSize="25sp"
				/>
</LinearLayout>
```

then, I initialize the button text in fragment:

```java
@Override
public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
	View rootView = inflater.inflate(R.layout.fragment_rate, container, false);
	BootstrapButton buttonGoodRate = (BootstrapButton) rootView.findViewById(R.id.fragment_rate_button_good);
	buttonBadRate.setText("GOOD");
	return rootView;
}
```

In case if I use XML **android:text** attribute everything works fine.
The issue occurs only in case of setting the text programmatically

Any workaround?

Thank you

**Comment by jamie-beardedhen** [comment]
The last code snippet seems to indicate that the text setter is being called on a different button from the one defined in your layout. If you can still reproduce this issue, it would be helpful to know what version of Android you're using.

**Comment by shell-software** [comment]
Hi,

Thank you for reply.

This is a "copy-paste" error (when I copied the text I removed several lines to make it more simple for you to read).
In my case I have to bootstrap buttons: buttonGoodRate and buttonBadRate. Setting the text via XML attribute works fine, but when I try to set it programmatically it is not visible (however, calling getText returns the set text value)

I am using emulator 4.0.3 ICS x86 image and the Library version 1.2.3

Could you please check whether *setText(String)* works correctly from your side and I will dig deeper to find the cause

**Comment by jamie-beardedhen** [comment]
Fixed in 2.0.0
